src/Modal.jsx

```diff
@@ -15,6 +18,22 @@ class Modal extends React.Component {
 		super(props);
 		this.onDismiss = this.onDismiss.bind(this);
 		this.onKeyDown = this.onKeyDown.bind(this);
+		this.getModalPosition = this.getModalPosition.bind(this);
+
+		this.state = {
+			topPosition: DEFAULT_MARGIN_TOP // matches default margin-top in CSS
+		};
+
+		if (!this.props.fullscreen && typeof window.matchMedia != 'undefined') {
```

Comment:
this is going to crash on the server where `window` is not defined - I recommend putting it in `componentDidMount`, which will only run in the browser